OpenOversight/app/models.py

```diff
@@ -37,6 +37,22 @@ def __repr__(self):
         return '<Department ID {}: {}>'.format(self.id, self.name)
 
 
+class Rank(db.Model):
+    __tablename__ = 'ranks'
+
+    id = db.Column(db.Integer, primary_key=True)
+    rank = db.Column(db.String(255), index=True, unique=False, nullable=False)
+    order = db.Column(db.Integer, index=True, unique=False, nullable=False)
+    department_id = db.Column(db.Integer, db.ForeignKey('departments.id'))
+    department = db.relationship('Department', backref='ranks')
```

Comment:
I think we want a unique constraint on `rank` name and `department_id` here, right now we can add duplicate ranks for the same department (see `Face` table)